source/utils/animationFrame.js

```diff
@@ -5,7 +5,7 @@ type CancelAnimationFrame = (requestId: number) => void;
 type RequestAnimationFrame = (callback: Callback) => number;
 
 // Properly handle server-side rendering.
-let win;
+export let win;
```

Comment:
Instead of exporting win object and calling it with win, why you just don't want to export raf/caf with bound context `win`?

```
export const raf = request.bind(win);
export const caf = cancel.bind(win);
```

Reply:
Good call @steppefox, I can change that. I do think that it happens in the commonjs build too though; we're using webpack 1 currently and support for ES modules wasn't added until webpack 2.

Reply:
@bmakuh Btw, please try to provide some example in demo repository if you have a time of course, because it really looks strange and guys won't approve it then. Or i'll do than on  weekends if you want, cuz I already have some demo app, need to change import of react-virtualized to ES here: https://github.com/steppefox/sandbox-react-contactlist